spot-contracts/contracts/RouterV2.sol

```diff
@@ -57,6 +58,10 @@ contract RouterV2 {
             perp.updateState();
         }
 
+        if (bond != perp.getDepositBond()) {
```

Comment:
If we're doing a read anyway, seems there's no need to pass it in as a parameter.

WDYT about removing the `bond` parameter, then setting the function var to `perp.getDepositBond()`? Then we wouldn't need the revert condition either.

Reply:
The reason we pass in the parameter is to ensure the user know which bond he is depositing into (since the deposit bond can diverge between the time the user reads and the user executes).

This check here is redundant as perp would revert if you try to mint using a non-deposit tranche.

Reply:
It is an expensive read (updates state) to have since redundant

Reply:
Sure, we can keep as-is